packages/omg-js-util/src/typedData.js

```diff
@@ -0,0 +1,88 @@
+
+const NULL_ADDRESS = '0x0000000000000000000000000000000000000000'
+const NULL_INPUT = { blknum: 0, txindex: 0, oindex: 0 }
+const NULL_OUTPUT = { owner: NULL_ADDRESS, currency: NULL_ADDRESS, amount: 0 }
+const NULL_METADATA = '0x0000000000000000000000000000000000000000000000000000000000000000'
+
+const domainSpec = [
+  { name: 'name', type: 'string' },
+  { name: 'version', type: 'string' },
+  { name: 'verifyingContract', type: 'address' },
+  { name: 'salt', type: 'bytes32' }
+]
+
+const txSpec = [
+  { name: 'input0', type: 'Input' },
+  { name: 'input1', type: 'Input' },
+  { name: 'input2', type: 'Input' },
+  { name: 'input3', type: 'Input' },
+  { name: 'output0', type: 'Output' },
+  { name: 'output1', type: 'Output' },
+  { name: 'output2', type: 'Output' },
+  { name: 'output3', type: 'Output' },
+  { name: 'metadata', type: 'bytes32' }
+]
+
+const inputSpec = [
+  { name: 'blknum', type: 'uint256' },
+  { name: 'txindex', type: 'uint256' },
+  { name: 'oindex', type: 'uint256' }
+]
+
+const outputSpec = [
+  { name: 'owner', type: 'address' },
+  { name: 'currency', type: 'address' },
+  { name: 'amount', type: 'uint256' }
+]
+
+const domainData = {
+  name: 'OMG Network',
+  version: '1',
+  verifyingContract: '0x44de0ec539b8c4a4b530c78620fe8320167f2f74',
```

Comment:
Are we hardcoding this contract address to `Ari` ? Would that affect our transaction signatures on different environments/rootchain contracts ?

Reply:
You're right - I forgot to delete this. It gets overwritten later anyway so it doesn't really matter, but it obviously should not be there.